core/src/main/java/org/apache/carbondata/core/locks/CarbonLockFactory.java

```diff
@@ -105,4 +120,10 @@ private static void getLockTypeConfigured() {
     LOGGER.info("Configured lock type is: " + lockTypeConfigured);
   }
 
+  private static void getLockpath() {
+    lockPath = CarbonProperties.getInstance()
+        .getProperty(CarbonCommonConstants.LOCK_PATH, "")
+        .toUpperCase();
```

Comment:
Remove uppercase as it does not work in unix

Reply:
done